scripts/releaseSteps.js

```diff
@@ -19,10 +19,11 @@ function srcDirectory(item) {
   return path.join(__dirname, '..', 'packages', item, 'src');
 }
 
+// The order of these packages is important!
```

Comment:
Can you make more explicit that tokens should precede Gestalt due to the dependency?

`'gestalt-design-tokens' must precede 'gestalt'`